src/sass/rich_tooltip.scss

```
@@ -0,0 +1,22 @@
+body.is-dark .rich-tooltip-table .p-icon--status-queued-small {
+  @include vf-icon-status-small($colors--light-theme--icon);
+}
+
+.rich-tooltip-table .p-icon--status-queued-small {
+  @include vf-icon-status-small($colors--dark-theme--icon);
+}
+
+.rich-tooltip-table {
+  margin: 0;
+  min-height: 400px;
+  width: 425px;
+
+  td {
+    max-width: 300px;
+    width: 300px;
+  }
+
+  th {
```

```suggestion
  .rich-tooltip-value {
    max-width: 300px;
    width: 300px;
  }

  .rich-tooltip-heading {
```
Don't target elements without a class. Better introduce a custom class for it. And for naming best use something close to the [BEM / Block Element Modifier](https://en.bem.info/methodology/quick-start/) methodology.